viam-cartographer/src/carto_facade/carto_facade.h

```diff
@@ -77,12 +79,6 @@ typedef struct viam_carto_sensor_reading {
     uint64_t sensor_reading_time_unix_micro;
 } viam_carto_sensor_reading;
 
-typedef enum viam_carto_MODE {
```

Comment:
I'm removing this as I realized that the API of SLAM is that the mode is defined by the state of the data_dir & whether or not map_rate_sec is zero. As a result, this type & the config parameter doesn't need to exist.